Octokit/Models/Request/CollaboratorRequest.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Diagnostics;
+using System.Globalization;
+
+namespace Octokit
+{
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
+    public class CollaboratorRequest
+    {
+        public CollaboratorRequest()
+        {
+            Permission = Permission.Push;
```

Comment:
In the [api](https://developer.github.com/v3/repos/collaborators/#add-user-as-a-collaborator) the default value is push. So i thought it would be correct.